pkg/query-service/app/http_handler.go

```diff
@@ -4554,96 +4501,58 @@ func (aH *APIHandler) queryRangeV3(ctx context.Context, queryRangeParams *v3.Que
-							"aggregateOperator":     queryInfoResult.AggregateOperator,
-							"aggregateAttributeKey": queryInfoResult.AggregateAttributeKey,
-							"filterApplied":         queryInfoResult.FilterApplied,
-							"dashboardId":           dashboardID,
-							"widgetId":              widgetID,
-						}, claims.Email, true, false)
-					}
-					if alertMatched {
-						var alertID string
-						var alertIDMatch []string
-						alertIDRegex, err := regexp.Compile(`ruleId=(\d+)`)
-						if err != nil {
-							zap.S().Errorf("error while matching the alertIDRegex: %v", err)
-						} else {
-							alertIDMatch = alertIDRegex.FindStringSubmatch(referrer)
-						}
-
-						if len(alertIDMatch) > 1 {
-							alertID = alertIDMatch[1]
-						}
-						telemetry.GetInstance().SendEvent(telemetry.TELEMETRY_EVENT_SUCCESSFUL_ALERT_QUERY, map[string]interface{}{
-							"queryType":             queryRangeParams.CompositeQuery.QueryType,
-							"panelType":             queryRangeParams.CompositeQuery.PanelType,
-							"tracesUsed":            queryInfoResult.TracesUsed,
-							"logsUsed":              queryInfoResult.LogsUsed,
-							"metricsUsed":           queryInfoResult.MetricsUsed,
-							"numberOfQueries":       queryInfoResult.NumberOfQueries,
-							"groupByApplied":        queryInfoResult.GroupByApplied,
-							"aggregateOperator":     queryInfoResult.AggregateOperator,
-							"aggregateAttributeKey": queryInfoResult.AggregateAttributeKey,
-							"filterApplied":         queryInfoResult.FilterApplied,
-							"alertId":               alertID,
-						}, claims.Email, true, false)
-					}
-				}
+	if matched, _ := regexp.MatchString(`/dashboard/[a-zA-Z0-9\-]+/(new|edit)(?:\?.*)?$`, referrer); matched {
+		properties := queryInfoResult.ToMap()
+
+		if dashboardIDRegex, err := regexp.Compile(`/dashboard/([a-f0-9\-]+)/`); err == nil {
+			if matches := dashboardIDRegex.FindStringSubmatch(referrer); len(matches) > 1 {
+				properties["dashboard_id"] = matches[1]
 			}
 		}
+
+		if widgetIDRegex, err := regexp.Compile(`widgetId=([a-f0-9\-]+)`); err == nil {
+			if matches := widgetIDRegex.FindStringSubmatch(referrer); len(matches) > 1 {
+				properties["widget_id"] = matches[1]
+			}
+		}
+
+		properties["referrer"] = referrer
+		properties["module_name"] = "dashboard"
+		aH.Signoz.Analytics.TrackUser(r.Context(), claims.OrgID, claims.UserID, "Telemetry Queried", properties)
 	}
+
+	if matched, _ := regexp.MatchString(`/alerts/(new|edit)(?:\?.*)?$`, referrer); matched {
+		properties := queryInfoResult.ToMap()
+
+		if alertIDRegex, err := regexp.Compile(`ruleId=(\d+)`); err == nil {
+			if matches := alertIDRegex.FindStringSubmatch(referrer); len(matches) > 1 {
+				properties["alert_id"] = matches[1]
+			}
+		}
+
+		properties["referrer"] = referrer
+		properties["module_name"] = "rule"
+		aH.Signoz.Analytics.TrackUser(r.Context(), claims.OrgID, claims.UserID, "Telemetry Queried", properties)
```

Comment:
Using this event we can't know if user actually saw the data on UI or not so this event is not a replacement for `TELEMETRY_EVENT_SUCCESSFUL_DASHBOARD_PANEL_QUERY` or `TELEMETRY_EVENT_SUCCESSFUL_ALERT_QUERY`